extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js

```diff
@@ -67,6 +68,18 @@ define('ext.wikia.adEngine.adInfoTrackerHelper',  [
 		return data;
 	}
 
+	function transformBidderPrice(bidderName, realSlotPrices, slotPricesIgnoringTimeout) {
+		if (realSlotPrices[bidderName]) {
```

Comment:
price comes as a string so it will be casted as bool:
`!!'0.00' // true` so I believe it'll be tracked correctly.